fuzz_test.go

```diff
@@ -192,8 +194,8 @@ func fuzzUnmarshal(data []byte) int {
 	}
 
 	for _, t := range types {
-		unmarshal(&buffer{b: data}, t)
-		readAny(&buffer{b: data})
+		_ = unmarshal(&buffer{b: data}, t)
```

Comment:
This got me looking closer at these fuzz tests.  While they exercise code-paths, there's nothing that actually triggers the tests to fail.  I opened https://github.com/Azure/go-amqp/issues/53 to track this.